### PR TITLE
chore: modernize frontcontrollers to match symfony 5.x

### PR DESCRIPTION
**Pull request by mussbach**
Reimplement changes of https://github.com/symfony/recipes/pull/724/files to our customized frontcontrollers

### How to review/test
Browsercalls as well as commands should work as before

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Application/FrontController.php [hunk 1]
```diff
@@ -76,21 +58,41 @@ public static function bootstrap(): void
...
+            echo 'Warning: The console should be invoked via the CLI version of PHP, not the '.PHP_SAPI.' SAPI'.PHP_EOL;
+        }
+
         set_time_limit(0);
 
+        require DemosPlanPath::getRootPath('vendor/autoload.php');
+
+        if (!class_exists(ConsoleApplication::class) || !class_exists(Dotenv::class)) {
+            throw new LogicException('You need to add "symfony/framework-bundle" and "symfony/dotenv" as Composer dependencies.');
+        }
```
Comment (graupnerdemos) [inline demosplan/DemosPlanCoreBundle/Application/FrontController.php:71]: We technically don't need this as this FrontController should not be available outside of a demosplan-core with all dependencies installed. However, we should keep it to make diffing against the Symfony baseline easier.

**Review by graupnerdemos — COMMENTED** [review]
Is there any specific reason why you opted to manually load DotEnv and the Addon registry in `console` but used `bootstrap` in `web`? 

I can't quite decide what the best solution might be here. Maybe, we rename `bootstrap` to `dplanInit` and – for now – only leave the Addon registration in there. That way, the front controller functions could remain more or less identical to the official ones except for this one extra method to be called.

File: demosplan/DemosPlanCoreBundle/Application/FrontController.php (hunk 1 above)
Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Application/FrontController.php:71]: this is exactly my intention to keep it here

**Comment by mussbach** [comment]
Bootstrap is used twice and I did not want to duplicate the logic

**Comment by graupnerdemos** [comment]
Ah, I see, it's also used in the PHPUnit bootstrap. In that case, can we not also use it in `web()`? Doesn't change logic, but is a nice repetition reduction refactoring.

**Review by graupnerdemos — APPROVED** [review]
lgtm. thanks.

**Pull request merged**